apple/sensor/ReanimatedSensor.h

```diff
@@ -1,4 +1,4 @@
-#if __has_include(<CoreMotion/CoreMotion.h>)
+#if __has_include(<CoreMotion/CoreMotion.h>) && !TARGET_OS_OSX
```

Comment:
Do you think we could use `#if !TARGET_OS_TV && !TARGET_OS_OSX` here as well for consistency?